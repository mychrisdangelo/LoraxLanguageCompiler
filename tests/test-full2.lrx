/* 
 * Author: Chris D'Angelo
 * End to end test of assignment to all types
 */

int main()
{

	int a;
	float b;
	bool c;
	char d;
	a = 1;
	b = 3.14;
	c = true;
	d = 'a';
	print(a, b, c, d);
}