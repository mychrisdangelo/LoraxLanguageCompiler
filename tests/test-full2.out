13.140000truea